[01-19 06:36:53]: [INFO] Created column 'regdate' and set the current timestamp, 1737268613884, to all 0 rows
[01-19 06:36:55]: [INFO] SQLite Setup finished
[01-19 06:36:55]: [INFO] No supported permissions system found! Permissions are disabled!
[01-19 06:36:55]: [INFO] AuthMe 5.6.0 build n.CUSTOM successfully enabled!
[01-19 06:36:56]: [INFO] Downloading GEO IP database, because the old database is older than 30 days or doesn't exist
[01-19 06:36:56]: [WARN] No MaxMind credentials found in the configuration file! GeoIp protections will be disabled.
[01-19 06:36:56]: [INFO] There is no newer GEO IP database uploaded to MaxMind. Using the old one for now.
[01-19 06:36:56]: [WARN] Could not download GeoLiteAPI database [FileNotFoundException]: plugins/AuthMe/GeoLite2-Country.mmdb (No such file or directory)
[01-19 06:36:56]: java.io.FileNotFoundException: plugins/AuthMe/GeoLite2-Country.mmdb (No such file or directory)
	at java.base/java.io.RandomAccessFile.open0(Native Method)
	at java.base/java.io.RandomAccessFile.open(RandomAccessFile.java:356)
	at java.base/java.io.RandomAccessFile.<init>(RandomAccessFile.java:273)
	at java.base/java.io.RandomAccessFile.<init>(RandomAccessFile.java:223)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.maxmind.db.BufferHolder.<init>(BufferHolder.java:20)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.maxmind.db.Reader.<init>(Reader.java:121)
	at AuthMe-5.6.0.jar//fr.xephi.authme.service.GeoIpService.startReading(GeoIpService.java:179)
	at AuthMe-5.6.0.jar//fr.xephi.authme.service.GeoIpService.updateDatabase(GeoIpService.java:150)
	at org.bukkit.craftbukkit.scheduler.CraftTask.run(CraftTask.java:78)
	at org.bukkit.craftbukkit.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:57)
	at com.destroystokyo.paper.ServerSchedulerReportingWrapper.run(ServerSchedulerReportingWrapper.java:22)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)

[01-19 06:57:55]: [INFO] AuthMe 5.6.0-bCUSTOM disabled!
[01-19 07:38:57]: [INFO] Connection arguments loaded, Hikari ConnectionPool ready!
[01-19 07:38:59]: [INFO] MySQL setup finished
[01-19 07:38:59]: [INFO] No supported permissions system found! Permissions are disabled!
[01-19 07:38:59]: [INFO] AuthMe 5.6.0 build n.CUSTOM successfully enabled!
[01-19 07:45:04]: [INFO] AuthMe 5.6.0-bCUSTOM disabled!
[01-19 07:45:35]: [INFO] Connection arguments loaded, Hikari ConnectionPool ready!
[01-19 07:45:36]: [INFO] MySQL setup finished
[01-19 07:45:36]: [INFO] No supported permissions system found! Permissions are disabled!
[01-19 07:45:36]: [INFO] AuthMe 5.6.0 build n.CUSTOM successfully enabled!
[01-19 08:31:47]: [INFO] AuthMe 5.6.0-bCUSTOM disabled!
[01-19 08:33:06]: [INFO] Connection arguments loaded, Hikari ConnectionPool ready!
[01-19 08:33:07]: [INFO] MySQL setup finished
[01-19 08:33:07]: [INFO] No supported permissions system found! Permissions are disabled!
[01-19 08:33:08]: [INFO] AuthMe 5.6.0 build n.CUSTOM successfully enabled!
[01-19 09:00:53]: [INFO] AuthMe 5.6.0-bCUSTOM disabled!
[01-19 12:16:03]: [INFO] Connection arguments loaded, Hikari ConnectionPool ready!
[01-19 12:16:03]: [INFO] MySQL setup finished
[01-19 12:16:04]: [INFO] No supported permissions system found! Permissions are disabled!
[01-19 12:16:04]: [INFO] AuthMe 5.6.0 build n.CUSTOM successfully enabled!
[01-19 12:19:14]: [WARN] Error during SQL operation: [SQLSyntaxErrorException]: (conn=51) Data too long for column 'password' at row 1
[01-19 12:19:14]: java.sql.SQLSyntaxErrorException: (conn=51) Data too long for column 'password' at row 1
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.export.ExceptionFactory.createException(ExceptionFactory.java:289)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.export.ExceptionFactory.create(ExceptionFactory.java:378)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.message.ClientMessage.readPacket(ClientMessage.java:172)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.client.impl.StandardClient.readPacket(StandardClient.java:886)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.client.impl.StandardClient.readResults(StandardClient.java:825)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.client.impl.StandardClient.readResponse(StandardClient.java:744)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.client.impl.StandardClient.execute(StandardClient.java:668)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.ClientPreparedStatement.executeInternal(ClientPreparedStatement.java:91)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.ClientPreparedStatement.executeLargeUpdate(ClientPreparedStatement.java:336)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.ClientPreparedStatement.executeUpdate(ClientPreparedStatement.java:313)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.zaxxer.hikari.pool.ProxyPreparedStatement.executeUpdate(ProxyPreparedStatement.java:61)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.zaxxer.hikari.pool.HikariProxyPreparedStatement.executeUpdate(HikariProxyPreparedStatement.java)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.ch.jalu.datasourcecolumns.sqlimplementation.SqlColumnsHandler.performUpdateAction(SqlColumnsHandler.java:338)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.ch.jalu.datasourcecolumns.sqlimplementation.SqlColumnsHandler.performInsert(SqlColumnsHandler.java:285)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.ch.jalu.datasourcecolumns.sqlimplementation.SqlColumnsHandler.insert(SqlColumnsHandler.java:199)
	at AuthMe-5.6.0.jar//fr.xephi.authme.datasource.columnshandler.AuthMeColumnsHandler.insert(AuthMeColumnsHandler.java:186)
	at AuthMe-5.6.0.jar//fr.xephi.authme.datasource.AbstractSqlDataSource.saveAuth(AbstractSqlDataSource.java:54)
	at AuthMe-5.6.0.jar//fr.xephi.authme.datasource.MySQL.saveAuth(MySQL.java:330)
	at AuthMe-5.6.0.jar//fr.xephi.authme.process.register.AsyncRegister.executeRegistration(AsyncRegister.java:102)
	at AuthMe-5.6.0.jar//fr.xephi.authme.process.register.AsyncRegister.register(AsyncRegister.java:57)
	at AuthMe-5.6.0.jar//fr.xephi.authme.process.Management.lambda$performRegister$4(Management.java:69)
	at org.bukkit.craftbukkit.scheduler.CraftTask.run(CraftTask.java:78)
	at org.bukkit.craftbukkit.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:57)
	at com.destroystokyo.paper.ServerSchedulerReportingWrapper.run(ServerSchedulerReportingWrapper.java:22)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)

[01-19 12:19:14]: [FINE] Gimerbone registered 192.168.100.105
[01-19 12:21:20]: [WARN] Error during SQL operation: [SQLSyntaxErrorException]: (conn=51) Data too long for column 'password' at row 1
[01-19 12:21:20]: java.sql.SQLSyntaxErrorException: (conn=51) Data too long for column 'password' at row 1
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.export.ExceptionFactory.createException(ExceptionFactory.java:289)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.export.ExceptionFactory.create(ExceptionFactory.java:378)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.message.ClientMessage.readPacket(ClientMessage.java:172)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.client.impl.StandardClient.readPacket(StandardClient.java:886)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.client.impl.StandardClient.readResults(StandardClient.java:825)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.client.impl.StandardClient.readResponse(StandardClient.java:744)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.client.impl.StandardClient.execute(StandardClient.java:668)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.ClientPreparedStatement.executeInternal(ClientPreparedStatement.java:91)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.ClientPreparedStatement.executeLargeUpdate(ClientPreparedStatement.java:336)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.ClientPreparedStatement.executeUpdate(ClientPreparedStatement.java:313)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.zaxxer.hikari.pool.ProxyPreparedStatement.executeUpdate(ProxyPreparedStatement.java:61)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.zaxxer.hikari.pool.HikariProxyPreparedStatement.executeUpdate(HikariProxyPreparedStatement.java)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.ch.jalu.datasourcecolumns.sqlimplementation.SqlColumnsHandler.performUpdateAction(SqlColumnsHandler.java:338)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.ch.jalu.datasourcecolumns.sqlimplementation.SqlColumnsHandler.performInsert(SqlColumnsHandler.java:285)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.ch.jalu.datasourcecolumns.sqlimplementation.SqlColumnsHandler.insert(SqlColumnsHandler.java:199)
	at AuthMe-5.6.0.jar//fr.xephi.authme.datasource.columnshandler.AuthMeColumnsHandler.insert(AuthMeColumnsHandler.java:186)
	at AuthMe-5.6.0.jar//fr.xephi.authme.datasource.AbstractSqlDataSource.saveAuth(AbstractSqlDataSource.java:54)
	at AuthMe-5.6.0.jar//fr.xephi.authme.datasource.MySQL.saveAuth(MySQL.java:330)
	at AuthMe-5.6.0.jar//fr.xephi.authme.process.register.AsyncRegister.executeRegistration(AsyncRegister.java:102)
	at AuthMe-5.6.0.jar//fr.xephi.authme.process.register.AsyncRegister.register(AsyncRegister.java:57)
	at AuthMe-5.6.0.jar//fr.xephi.authme.process.Management.lambda$performRegister$4(Management.java:69)
	at org.bukkit.craftbukkit.scheduler.CraftTask.run(CraftTask.java:78)
	at org.bukkit.craftbukkit.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:57)
	at com.destroystokyo.paper.ServerSchedulerReportingWrapper.run(ServerSchedulerReportingWrapper.java:22)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)

[01-19 12:21:20]: [FINE] Gimerbone registered 192.168.100.105
[01-19 13:27:01]: [INFO] AuthMe 5.6.0-bCUSTOM disabled!
[01-19 13:27:57]: [INFO] Connection arguments loaded, Hikari ConnectionPool ready!
[01-19 13:27:57]: [INFO] MySQL setup finished
[01-19 13:27:57]: [INFO] No supported permissions system found! Permissions are disabled!
[01-19 13:27:57]: [INFO] AuthMe 5.6.0 build n.CUSTOM successfully enabled!
[01-19 13:32:13]: [WARN] Error during SQL operation: [SQLSyntaxErrorException]: (conn=1523) Data too long for column 'password' at row 1
[01-19 13:32:13]: java.sql.SQLSyntaxErrorException: (conn=1523) Data too long for column 'password' at row 1
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.export.ExceptionFactory.createException(ExceptionFactory.java:289)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.export.ExceptionFactory.create(ExceptionFactory.java:378)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.message.ClientMessage.readPacket(ClientMessage.java:172)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.client.impl.StandardClient.readPacket(StandardClient.java:886)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.client.impl.StandardClient.readResults(StandardClient.java:825)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.client.impl.StandardClient.readResponse(StandardClient.java:744)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.client.impl.StandardClient.execute(StandardClient.java:668)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.ClientPreparedStatement.executeInternal(ClientPreparedStatement.java:91)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.ClientPreparedStatement.executeLargeUpdate(ClientPreparedStatement.java:336)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.ClientPreparedStatement.executeUpdate(ClientPreparedStatement.java:313)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.zaxxer.hikari.pool.ProxyPreparedStatement.executeUpdate(ProxyPreparedStatement.java:61)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.zaxxer.hikari.pool.HikariProxyPreparedStatement.executeUpdate(HikariProxyPreparedStatement.java)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.ch.jalu.datasourcecolumns.sqlimplementation.SqlColumnsHandler.performUpdateAction(SqlColumnsHandler.java:338)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.ch.jalu.datasourcecolumns.sqlimplementation.SqlColumnsHandler.performInsert(SqlColumnsHandler.java:285)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.ch.jalu.datasourcecolumns.sqlimplementation.SqlColumnsHandler.insert(SqlColumnsHandler.java:199)
	at AuthMe-5.6.0.jar//fr.xephi.authme.datasource.columnshandler.AuthMeColumnsHandler.insert(AuthMeColumnsHandler.java:186)
	at AuthMe-5.6.0.jar//fr.xephi.authme.datasource.AbstractSqlDataSource.saveAuth(AbstractSqlDataSource.java:54)
	at AuthMe-5.6.0.jar//fr.xephi.authme.datasource.MySQL.saveAuth(MySQL.java:330)
	at AuthMe-5.6.0.jar//fr.xephi.authme.process.register.AsyncRegister.executeRegistration(AsyncRegister.java:102)
	at AuthMe-5.6.0.jar//fr.xephi.authme.process.register.AsyncRegister.register(AsyncRegister.java:57)
	at AuthMe-5.6.0.jar//fr.xephi.authme.process.Management.lambda$performRegister$4(Management.java:69)
	at org.bukkit.craftbukkit.scheduler.CraftTask.run(CraftTask.java:78)
	at org.bukkit.craftbukkit.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:57)
	at com.destroystokyo.paper.ServerSchedulerReportingWrapper.run(ServerSchedulerReportingWrapper.java:22)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)

[01-19 13:32:13]: [FINE] Gimerbone registered 192.168.100.105
[01-19 13:32:55]: [WARN] Error during SQL operation: [SQLSyntaxErrorException]: (conn=1523) Data too long for column 'password' at row 1
[01-19 13:32:55]: java.sql.SQLSyntaxErrorException: (conn=1523) Data too long for column 'password' at row 1
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.export.ExceptionFactory.createException(ExceptionFactory.java:289)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.export.ExceptionFactory.create(ExceptionFactory.java:378)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.message.ClientMessage.readPacket(ClientMessage.java:172)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.client.impl.StandardClient.readPacket(StandardClient.java:886)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.client.impl.StandardClient.readResults(StandardClient.java:825)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.client.impl.StandardClient.readResponse(StandardClient.java:744)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.client.impl.StandardClient.execute(StandardClient.java:668)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.ClientPreparedStatement.executeInternal(ClientPreparedStatement.java:91)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.ClientPreparedStatement.executeLargeUpdate(ClientPreparedStatement.java:336)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.ClientPreparedStatement.executeUpdate(ClientPreparedStatement.java:313)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.zaxxer.hikari.pool.ProxyPreparedStatement.executeUpdate(ProxyPreparedStatement.java:61)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.zaxxer.hikari.pool.HikariProxyPreparedStatement.executeUpdate(HikariProxyPreparedStatement.java)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.ch.jalu.datasourcecolumns.sqlimplementation.SqlColumnsHandler.performUpdateAction(SqlColumnsHandler.java:338)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.ch.jalu.datasourcecolumns.sqlimplementation.SqlColumnsHandler.performInsert(SqlColumnsHandler.java:285)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.ch.jalu.datasourcecolumns.sqlimplementation.SqlColumnsHandler.insert(SqlColumnsHandler.java:199)
	at AuthMe-5.6.0.jar//fr.xephi.authme.datasource.columnshandler.AuthMeColumnsHandler.insert(AuthMeColumnsHandler.java:186)
	at AuthMe-5.6.0.jar//fr.xephi.authme.datasource.AbstractSqlDataSource.saveAuth(AbstractSqlDataSource.java:54)
	at AuthMe-5.6.0.jar//fr.xephi.authme.datasource.MySQL.saveAuth(MySQL.java:330)
	at AuthMe-5.6.0.jar//fr.xephi.authme.process.register.AsyncRegister.executeRegistration(AsyncRegister.java:102)
	at AuthMe-5.6.0.jar//fr.xephi.authme.process.register.AsyncRegister.register(AsyncRegister.java:57)
	at AuthMe-5.6.0.jar//fr.xephi.authme.process.Management.lambda$performRegister$4(Management.java:69)
	at org.bukkit.craftbukkit.scheduler.CraftTask.run(CraftTask.java:78)
	at org.bukkit.craftbukkit.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:57)
	at com.destroystokyo.paper.ServerSchedulerReportingWrapper.run(ServerSchedulerReportingWrapper.java:22)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)

[01-19 13:32:55]: [FINE] Gimerbone registered 192.168.100.105
[01-19 13:36:25]: [INFO] AuthMe 5.6.0-bCUSTOM disabled!
[01-19 13:37:14]: [INFO] Connection arguments loaded, Hikari ConnectionPool ready!
[01-19 13:37:15]: [INFO] MySQL setup finished
[01-19 13:37:15]: [INFO] No supported permissions system found! Permissions are disabled!
[01-19 13:37:16]: [INFO] AuthMe 5.6.0 build n.CUSTOM successfully enabled!
[01-19 13:38:08]: [WARN] Error during SQL operation: [SQLSyntaxErrorException]: (conn=32) Incorrect date value: '1737293888115' for column `authme`.`authme`.`registration_date` at row 1
[01-19 13:38:08]: java.sql.SQLSyntaxErrorException: (conn=32) Incorrect date value: '1737293888115' for column `authme`.`authme`.`registration_date` at row 1
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.export.ExceptionFactory.createException(ExceptionFactory.java:289)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.export.ExceptionFactory.create(ExceptionFactory.java:378)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.message.ClientMessage.readPacket(ClientMessage.java:172)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.client.impl.StandardClient.readPacket(StandardClient.java:886)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.client.impl.StandardClient.readResults(StandardClient.java:825)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.client.impl.StandardClient.readResponse(StandardClient.java:744)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.client.impl.StandardClient.execute(StandardClient.java:668)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.ClientPreparedStatement.executeInternal(ClientPreparedStatement.java:91)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.ClientPreparedStatement.executeLargeUpdate(ClientPreparedStatement.java:336)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.ClientPreparedStatement.executeUpdate(ClientPreparedStatement.java:313)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.zaxxer.hikari.pool.ProxyPreparedStatement.executeUpdate(ProxyPreparedStatement.java:61)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.zaxxer.hikari.pool.HikariProxyPreparedStatement.executeUpdate(HikariProxyPreparedStatement.java)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.ch.jalu.datasourcecolumns.sqlimplementation.SqlColumnsHandler.performUpdateAction(SqlColumnsHandler.java:338)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.ch.jalu.datasourcecolumns.sqlimplementation.SqlColumnsHandler.performInsert(SqlColumnsHandler.java:285)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.ch.jalu.datasourcecolumns.sqlimplementation.SqlColumnsHandler.insert(SqlColumnsHandler.java:199)
	at AuthMe-5.6.0.jar//fr.xephi.authme.datasource.columnshandler.AuthMeColumnsHandler.insert(AuthMeColumnsHandler.java:186)
	at AuthMe-5.6.0.jar//fr.xephi.authme.datasource.AbstractSqlDataSource.saveAuth(AbstractSqlDataSource.java:54)
	at AuthMe-5.6.0.jar//fr.xephi.authme.datasource.MySQL.saveAuth(MySQL.java:330)
	at AuthMe-5.6.0.jar//fr.xephi.authme.process.register.AsyncRegister.executeRegistration(AsyncRegister.java:102)
	at AuthMe-5.6.0.jar//fr.xephi.authme.process.register.AsyncRegister.register(AsyncRegister.java:57)
	at AuthMe-5.6.0.jar//fr.xephi.authme.process.Management.lambda$performRegister$4(Management.java:69)
	at org.bukkit.craftbukkit.scheduler.CraftTask.run(CraftTask.java:78)
	at org.bukkit.craftbukkit.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:57)
	at com.destroystokyo.paper.ServerSchedulerReportingWrapper.run(ServerSchedulerReportingWrapper.java:22)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)

[01-19 13:38:08]: [FINE] Gimerbone registered 192.168.100.105
[01-19 13:40:42]: [INFO] AuthMe 5.6.0-bCUSTOM disabled!
[01-19 13:41:09]: [INFO] Connection arguments loaded, Hikari ConnectionPool ready!
[01-19 13:41:10]: [INFO] MySQL setup finished
[01-19 13:41:10]: [INFO] No supported permissions system found! Permissions are disabled!
[01-19 13:41:10]: [INFO] AuthMe 5.6.0 build n.CUSTOM successfully enabled!
[01-19 13:41:54]: [FINE] Gimerbone registered 192.168.100.105
[01-19 13:41:54]: [WARN] Error during SQL operation: [SQLSyntaxErrorException]: (conn=31) Incorrect date value: '1737294114434' for column `authme`.`authme`.`last_login` at row 1
[01-19 13:41:54]: java.sql.SQLSyntaxErrorException: (conn=31) Incorrect date value: '1737294114434' for column `authme`.`authme`.`last_login` at row 1
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.export.ExceptionFactory.createException(ExceptionFactory.java:289)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.export.ExceptionFactory.create(ExceptionFactory.java:378)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.message.ClientMessage.readPacket(ClientMessage.java:172)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.client.impl.StandardClient.readPacket(StandardClient.java:886)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.client.impl.StandardClient.readResults(StandardClient.java:825)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.client.impl.StandardClient.readResponse(StandardClient.java:744)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.client.impl.StandardClient.execute(StandardClient.java:668)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.ClientPreparedStatement.executeInternal(ClientPreparedStatement.java:91)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.ClientPreparedStatement.executeLargeUpdate(ClientPreparedStatement.java:336)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.ClientPreparedStatement.executeUpdate(ClientPreparedStatement.java:313)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.zaxxer.hikari.pool.ProxyPreparedStatement.executeUpdate(ProxyPreparedStatement.java:61)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.zaxxer.hikari.pool.HikariProxyPreparedStatement.executeUpdate(HikariProxyPreparedStatement.java)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.ch.jalu.datasourcecolumns.sqlimplementation.SqlColumnsHandler.performUpdateAction(SqlColumnsHandler.java:338)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.ch.jalu.datasourcecolumns.sqlimplementation.SqlColumnsHandler.performUpdate(SqlColumnsHandler.java:232)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.ch.jalu.datasourcecolumns.sqlimplementation.SqlColumnsHandler.update(SqlColumnsHandler.java:175)
	at AuthMe-5.6.0.jar//fr.xephi.authme.datasource.columnshandler.AuthMeColumnsHandler.update(AuthMeColumnsHandler.java:98)
	at AuthMe-5.6.0.jar//fr.xephi.authme.datasource.AbstractSqlDataSource.updateSession(AbstractSqlDataSource.java:73)
	at AuthMe-5.6.0.jar//fr.xephi.authme.process.login.AsynchronousLogin.performLogin(AsynchronousLogin.java:277)
	at AuthMe-5.6.0.jar//fr.xephi.authme.process.login.AsynchronousLogin.forceLogin(AsynchronousLogin.java:118)
	at AuthMe-5.6.0.jar//fr.xephi.authme.process.register.executors.AbstractPasswordRegisterExecutor.lambda$executePostPersistAction$0(AbstractPasswordRegisterExecutor.java:92)
	at org.bukkit.craftbukkit.scheduler.CraftTask.run(CraftTask.java:78)
	at org.bukkit.craftbukkit.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:57)
	at com.destroystokyo.paper.ServerSchedulerReportingWrapper.run(ServerSchedulerReportingWrapper.java:22)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)

[01-19 13:41:54]: [FINE] Gimerbone logged in 192.168.100.105
[01-19 13:42:14]: [WARN] Error during SQL operation: [SQLSyntaxErrorException]: (conn=31) Incorrect date value: '1737294134826' for column `authme`.`authme`.`last_login` at row 1
[01-19 13:42:14]: java.sql.SQLSyntaxErrorException: (conn=31) Incorrect date value: '1737294134826' for column `authme`.`authme`.`last_login` at row 1
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.export.ExceptionFactory.createException(ExceptionFactory.java:289)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.export.ExceptionFactory.create(ExceptionFactory.java:378)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.message.ClientMessage.readPacket(ClientMessage.java:172)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.client.impl.StandardClient.readPacket(StandardClient.java:886)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.client.impl.StandardClient.readResults(StandardClient.java:825)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.client.impl.StandardClient.readResponse(StandardClient.java:744)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.client.impl.StandardClient.execute(StandardClient.java:668)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.ClientPreparedStatement.executeInternal(ClientPreparedStatement.java:91)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.ClientPreparedStatement.executeLargeUpdate(ClientPreparedStatement.java:336)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.ClientPreparedStatement.executeUpdate(ClientPreparedStatement.java:313)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.zaxxer.hikari.pool.ProxyPreparedStatement.executeUpdate(ProxyPreparedStatement.java:61)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.zaxxer.hikari.pool.HikariProxyPreparedStatement.executeUpdate(HikariProxyPreparedStatement.java)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.ch.jalu.datasourcecolumns.sqlimplementation.SqlColumnsHandler.performUpdateAction(SqlColumnsHandler.java:338)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.ch.jalu.datasourcecolumns.sqlimplementation.SqlColumnsHandler.performUpdate(SqlColumnsHandler.java:232)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.ch.jalu.datasourcecolumns.sqlimplementation.SqlColumnsHandler.update(SqlColumnsHandler.java:175)
	at AuthMe-5.6.0.jar//fr.xephi.authme.datasource.columnshandler.AuthMeColumnsHandler.update(AuthMeColumnsHandler.java:98)
	at AuthMe-5.6.0.jar//fr.xephi.authme.datasource.AbstractSqlDataSource.updateSession(AbstractSqlDataSource.java:73)
	at AuthMe-5.6.0.jar//fr.xephi.authme.process.quit.AsynchronousQuit.processQuit(AsynchronousQuit.java:86)
	at AuthMe-5.6.0.jar//fr.xephi.authme.process.Management.lambda$performQuit$8(Management.java:85)
	at org.bukkit.craftbukkit.scheduler.CraftTask.run(CraftTask.java:78)
	at org.bukkit.craftbukkit.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:57)
	at com.destroystokyo.paper.ServerSchedulerReportingWrapper.run(ServerSchedulerReportingWrapper.java:22)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)

[01-19 13:42:31]: [WARN] Error during SQL operation: [SQLSyntaxErrorException]: (conn=31) Incorrect date value: '1737294151633' for column `authme`.`authme`.`last_login` at row 1
[01-19 13:42:31]: java.sql.SQLSyntaxErrorException: (conn=31) Incorrect date value: '1737294151633' for column `authme`.`authme`.`last_login` at row 1
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.export.ExceptionFactory.createException(ExceptionFactory.java:289)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.export.ExceptionFactory.create(ExceptionFactory.java:378)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.message.ClientMessage.readPacket(ClientMessage.java:172)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.client.impl.StandardClient.readPacket(StandardClient.java:886)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.client.impl.StandardClient.readResults(StandardClient.java:825)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.client.impl.StandardClient.readResponse(StandardClient.java:744)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.client.impl.StandardClient.execute(StandardClient.java:668)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.ClientPreparedStatement.executeInternal(ClientPreparedStatement.java:91)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.ClientPreparedStatement.executeLargeUpdate(ClientPreparedStatement.java:336)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.ClientPreparedStatement.executeUpdate(ClientPreparedStatement.java:313)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.zaxxer.hikari.pool.ProxyPreparedStatement.executeUpdate(ProxyPreparedStatement.java:61)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.zaxxer.hikari.pool.HikariProxyPreparedStatement.executeUpdate(HikariProxyPreparedStatement.java)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.ch.jalu.datasourcecolumns.sqlimplementation.SqlColumnsHandler.performUpdateAction(SqlColumnsHandler.java:338)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.ch.jalu.datasourcecolumns.sqlimplementation.SqlColumnsHandler.performUpdate(SqlColumnsHandler.java:232)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.ch.jalu.datasourcecolumns.sqlimplementation.SqlColumnsHandler.update(SqlColumnsHandler.java:175)
	at AuthMe-5.6.0.jar//fr.xephi.authme.datasource.columnshandler.AuthMeColumnsHandler.update(AuthMeColumnsHandler.java:98)
	at AuthMe-5.6.0.jar//fr.xephi.authme.datasource.AbstractSqlDataSource.updateSession(AbstractSqlDataSource.java:73)
	at AuthMe-5.6.0.jar//fr.xephi.authme.process.login.AsynchronousLogin.performLogin(AsynchronousLogin.java:277)
	at AuthMe-5.6.0.jar//fr.xephi.authme.process.login.AsynchronousLogin.login(AsynchronousLogin.java:105)
	at AuthMe-5.6.0.jar//fr.xephi.authme.process.Management.lambda$performLogin$0(Management.java:53)
	at org.bukkit.craftbukkit.scheduler.CraftTask.run(CraftTask.java:78)
	at org.bukkit.craftbukkit.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:57)
	at com.destroystokyo.paper.ServerSchedulerReportingWrapper.run(ServerSchedulerReportingWrapper.java:22)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)

[01-19 13:42:31]: [FINE] Gimerbone logged in 192.168.100.105
[01-19 13:42:50]: [WARN] Error during SQL operation: [SQLSyntaxErrorException]: (conn=31) Incorrect date value: '1737294170934' for column `authme`.`authme`.`last_login` at row 1
[01-19 13:42:50]: java.sql.SQLSyntaxErrorException: (conn=31) Incorrect date value: '1737294170934' for column `authme`.`authme`.`last_login` at row 1
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.export.ExceptionFactory.createException(ExceptionFactory.java:289)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.export.ExceptionFactory.create(ExceptionFactory.java:378)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.message.ClientMessage.readPacket(ClientMessage.java:172)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.client.impl.StandardClient.readPacket(StandardClient.java:886)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.client.impl.StandardClient.readResults(StandardClient.java:825)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.client.impl.StandardClient.readResponse(StandardClient.java:744)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.client.impl.StandardClient.execute(StandardClient.java:668)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.ClientPreparedStatement.executeInternal(ClientPreparedStatement.java:91)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.ClientPreparedStatement.executeLargeUpdate(ClientPreparedStatement.java:336)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.org.mariadb.jdbc.ClientPreparedStatement.executeUpdate(ClientPreparedStatement.java:313)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.zaxxer.hikari.pool.ProxyPreparedStatement.executeUpdate(ProxyPreparedStatement.java:61)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.zaxxer.hikari.pool.HikariProxyPreparedStatement.executeUpdate(HikariProxyPreparedStatement.java)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.ch.jalu.datasourcecolumns.sqlimplementation.SqlColumnsHandler.performUpdateAction(SqlColumnsHandler.java:338)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.ch.jalu.datasourcecolumns.sqlimplementation.SqlColumnsHandler.performUpdate(SqlColumnsHandler.java:232)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.ch.jalu.datasourcecolumns.sqlimplementation.SqlColumnsHandler.update(SqlColumnsHandler.java:175)
	at AuthMe-5.6.0.jar//fr.xephi.authme.datasource.columnshandler.AuthMeColumnsHandler.update(AuthMeColumnsHandler.java:98)
	at AuthMe-5.6.0.jar//fr.xephi.authme.datasource.AbstractSqlDataSource.updateSession(AbstractSqlDataSource.java:73)
	at AuthMe-5.6.0.jar//fr.xephi.authme.process.quit.AsynchronousQuit.processQuit(AsynchronousQuit.java:86)
	at AuthMe-5.6.0.jar//fr.xephi.authme.process.Management.lambda$performQuit$8(Management.java:85)
	at org.bukkit.craftbukkit.scheduler.CraftTask.run(CraftTask.java:78)
	at org.bukkit.craftbukkit.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:57)
	at com.destroystokyo.paper.ServerSchedulerReportingWrapper.run(ServerSchedulerReportingWrapper.java:22)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)

[01-19 13:43:57]: [INFO] AuthMe 5.6.0-bCUSTOM disabled!
[01-19 13:54:33]: [INFO] Connection arguments loaded, Hikari ConnectionPool ready!
[01-19 13:54:33]: [INFO] MySQL setup finished
[01-19 13:54:33]: [INFO] No supported permissions system found! Permissions are disabled!
[01-19 13:54:34]: [WARN] Note: Hooks.bungeecord is set to true but your server appears to be running in non-bungeecord mode (see your spigot.yml). In order to prevent untrusted payload attack, BungeeCord hook will be automatically disabled!
[01-19 13:54:34]: [INFO] AuthMe 5.6.0 build n.CUSTOM successfully enabled!
[01-19 13:55:35]: [INFO] AuthMe 5.6.0-bCUSTOM disabled!
[01-19 13:55:53]: [INFO] Connection arguments loaded, Hikari ConnectionPool ready!
[01-19 13:55:54]: [INFO] MySQL setup finished
[01-19 13:55:54]: [INFO] No supported permissions system found! Permissions are disabled!
[01-19 13:55:55]: [INFO] AuthMe 5.6.0 build n.CUSTOM successfully enabled!
[01-19 13:57:41]: [INFO] AuthMe 5.6.0-bCUSTOM disabled!
[01-19 14:05:59]: [INFO] Connection arguments loaded, Hikari ConnectionPool ready!
[01-19 14:05:59]: [INFO] MySQL setup finished
[01-19 14:05:59]: [INFO] No supported permissions system found! Permissions are disabled!
[01-19 14:06:00]: [INFO] AuthMe 5.6.0 build n.CUSTOM successfully enabled!
[01-19 14:07:10]: [FINE] Gimerbone logged in 192.168.100.105
[01-19 14:09:05]: [FINE] Gimerbone logged in 192.168.100.105
[01-19 14:09:36]: [INFO] AuthMe 5.6.0-bCUSTOM disabled!
[01-19 14:11:03]: [INFO] Connection arguments loaded, Hikari ConnectionPool ready!
[01-19 14:11:04]: [INFO] MySQL setup finished
[01-19 14:11:04]: [INFO] No supported permissions system found! Permissions are disabled!
[01-19 14:11:04]: [INFO] AuthMe 5.6.0 build n.CUSTOM successfully enabled!
[01-19 14:12:12]: [FINE] Gimerbone logged in 192.168.100.105
[01-19 14:17:20]: [INFO] AuthMe 5.6.0-bCUSTOM disabled!
[01-19 14:17:38]: [INFO] Connection arguments loaded, Hikari ConnectionPool ready!
[01-19 14:17:39]: [INFO] MySQL setup finished
[01-19 14:17:39]: [INFO] No supported permissions system found! Permissions are disabled!
[01-19 14:17:39]: [INFO] AuthMe 5.6.0 build n.CUSTOM successfully enabled!
[01-19 14:21:39]: [FINE] Gimerbone logged in 192.168.100.105
[01-19 14:26:49]: [INFO] AuthMe 5.6.0-bCUSTOM disabled!
[01-19 14:27:07]: [INFO] Connection arguments loaded, Hikari ConnectionPool ready!
[01-19 14:27:07]: [INFO] MySQL setup finished
[01-19 14:27:07]: [INFO] No supported permissions system found! Permissions are disabled!
[01-19 14:27:08]: [INFO] AuthMe 5.6.0 build n.CUSTOM successfully enabled!
[01-19 14:29:46]: [FINE] Gimerbone logged in 192.168.100.105
[01-19 14:30:08]: [FINE] Gimerbone logged in 192.168.100.105
[01-19 14:32:29]: [FINE] Gimerbone logged in 192.168.100.105
[01-19 14:44:10]: [INFO] AuthMe 5.6.0-bCUSTOM disabled!
[01-19 14:45:08]: [INFO] Connection arguments loaded, Hikari ConnectionPool ready!
[01-19 14:45:08]: [INFO] MySQL setup finished
[01-19 14:45:08]: [INFO] No supported permissions system found! Permissions are disabled!
[01-19 14:45:08]: [INFO] AuthMe 5.6.0 build n.CUSTOM successfully enabled!
[01-19 14:47:03]: [FINE] Gimerbone logged in 192.168.100.105
[01-19 14:47:10]: [FINE] Gimerbone logged in 192.168.100.105
[01-19 14:49:08]: [INFO] AuthMe 5.6.0-bCUSTOM disabled!
[01-19 14:49:25]: [INFO] Connection arguments loaded, Hikari ConnectionPool ready!
[01-19 14:49:26]: [INFO] MySQL setup finished
[01-19 14:49:26]: [INFO] No supported permissions system found! Permissions are disabled!
[01-19 14:49:26]: [INFO] AuthMe 5.6.0 build n.CUSTOM successfully enabled!
[01-19 14:50:09]: [FINE] Gimerbone logged in 192.168.100.105
[01-19 14:50:18]: [FINE] Gimerbone logged in 192.168.100.105
[01-19 17:40:00]: [INFO] AuthMe 5.6.0-bCUSTOM disabled!
[01-20 02:19:44]: [INFO] Connection arguments loaded, Hikari ConnectionPool ready!
[01-20 02:19:45]: [INFO] MySQL setup finished
[01-20 02:19:45]: [INFO] No supported permissions system found! Permissions are disabled!
[01-20 02:19:46]: [INFO] AuthMe 5.6.0 build n.CUSTOM successfully enabled!
[01-20 04:58:49]: [INFO] AuthMe 5.6.0-bCUSTOM disabled!
[01-20 04:59:10]: [INFO] Connection arguments loaded, Hikari ConnectionPool ready!
[01-20 04:59:10]: [INFO] MySQL setup finished
[01-20 04:59:10]: [INFO] No supported permissions system found! Permissions are disabled!
[01-20 04:59:11]: [INFO] AuthMe 5.6.0 build n.CUSTOM successfully enabled!
[01-20 05:00:20]: [FINE] EiRanGerZ registered 192.168.100.105
[01-20 05:00:20]: [FINE] The user EiRanGerZ has 2 accounts:
[01-20 05:00:20]: [FINE] §7gimerbone, §aEiRanGerZ§7.
[01-20 05:00:20]: [FINE] EiRanGerZ logged in 192.168.100.105
[01-20 06:27:57]: [FINE] yaminousagi registered 192.168.100.105
[01-20 06:27:57]: [FINE] The user yaminousagi has 3 accounts:
[01-20 06:27:57]: [FINE] §7gimerbone, eirangerz, §ayaminousagi§7.
[01-20 06:27:57]: [FINE] yaminousagi logged in 192.168.100.105
[01-20 06:31:56]: [FINE] Gimerbone used the wrong password
[01-20 06:32:09]: [FINE] The user Gimerbone has 3 accounts:
[01-20 06:32:09]: [FINE] §7§aGimerbone§7, eirangerz, yaminousagi.
[01-20 06:32:09]: [FINE] Gimerbone logged in 192.168.100.105
[01-20 06:33:53]: [FINE] The user yaminousagi has 3 accounts:
[01-20 06:33:53]: [FINE] §7gimerbone, eirangerz, §ayaminousagi§7.
[01-20 06:33:53]: [FINE] yaminousagi logged in 192.168.100.105
[01-20 06:44:23]: [FINE] The user Gimerbone has 3 accounts:
[01-20 06:44:23]: [FINE] §7§aGimerbone§7, eirangerz, yaminousagi.
[01-20 06:44:23]: [FINE] Gimerbone logged in 192.168.100.105
[01-20 06:44:23]: [FINE] The user yaminousagi has 3 accounts:
[01-20 06:44:23]: [FINE] §7§aGimerbone§7, eirangerz, §ayaminousagi§7.
[01-20 06:44:23]: [FINE] yaminousagi logged in 192.168.100.105
[01-20 06:44:37]: [INFO] AuthMe 5.6.0-bCUSTOM disabled!
[01-20 06:45:03]: [INFO] Connection arguments loaded, Hikari ConnectionPool ready!
[01-20 06:45:03]: [INFO] MySQL setup finished
[01-20 06:45:03]: [INFO] No supported permissions system found! Permissions are disabled!
[01-20 06:45:04]: [INFO] AuthMe 5.6.0 build n.CUSTOM successfully enabled!
[01-20 06:46:09]: [FINE] The user Gimerbone has 3 accounts:
[01-20 06:46:09]: [FINE] §7§aGimerbone§7, eirangerz, yaminousagi.
[01-20 06:46:09]: [FINE] Gimerbone logged in 192.168.100.105
[01-20 06:46:18]: [FINE] The user yaminousagi has 3 accounts:
[01-20 06:46:18]: [FINE] §7gimerbone, eirangerz, §ayaminousagi§7.
[01-20 06:46:18]: [FINE] yaminousagi logged in 192.168.100.105
[01-20 06:46:22]: [FINE] The user Gimerbone has 3 accounts:
[01-20 06:46:22]: [FINE] §7§aGimerbone§7, eirangerz, yaminousagi.
[01-20 06:46:22]: [FINE] Gimerbone logged in 192.168.100.105
[01-20 06:48:29]: [FINE] The user yaminousagi has 3 accounts:
[01-20 06:48:29]: [FINE] §7§aGimerbone§7, eirangerz, §ayaminousagi§7.
[01-20 06:48:29]: [FINE] yaminousagi logged in 192.168.100.105
[01-20 07:01:30]: [INFO] AuthMe 5.6.0-bCUSTOM disabled!
[01-20 07:01:46]: [INFO] Connection arguments loaded, Hikari ConnectionPool ready!
[01-20 07:01:47]: [INFO] MySQL setup finished
[01-20 07:01:47]: [INFO] No supported permissions system found! Permissions are disabled!
[01-20 07:01:47]: [INFO] AuthMe 5.6.0 build n.CUSTOM successfully enabled!
[01-20 07:05:16]: [INFO] AuthMe 5.6.0-bCUSTOM disabled!
[01-20 07:05:33]: [INFO] Connection arguments loaded, Hikari ConnectionPool ready!
[01-20 07:05:34]: [INFO] MySQL setup finished
[01-20 07:05:34]: [INFO] No supported permissions system found! Permissions are disabled!
[01-20 07:05:34]: [INFO] AuthMe 5.6.0 build n.CUSTOM successfully enabled!
[01-20 07:06:03]: [FINE] The user Gimerbone has 3 accounts:
[01-20 07:06:03]: [FINE] §7§aGimerbone§7, eirangerz, yaminousagi.
[01-20 07:06:03]: [FINE] Gimerbone logged in 192.168.100.105
[01-20 07:06:14]: [FINE] The user Gimerbone has 3 accounts:
[01-20 07:06:14]: [FINE] §7§aGimerbone§7, eirangerz, yaminousagi.
[01-20 07:06:14]: [FINE] Gimerbone logged in 192.168.100.105
[01-20 07:06:36]: [FINE] The user Gimerbone has 3 accounts:
[01-20 07:06:36]: [FINE] §7§aGimerbone§7, eirangerz, yaminousagi.
[01-20 07:06:36]: [FINE] Gimerbone logged in 192.168.100.105
[01-20 07:13:38]: [INFO] AuthMe 5.6.0-bCUSTOM disabled!
[01-20 07:13:55]: [INFO] Connection arguments loaded, Hikari ConnectionPool ready!
[01-20 07:13:55]: [INFO] MySQL setup finished
[01-20 07:13:55]: [INFO] No supported permissions system found! Permissions are disabled!
[01-20 07:13:56]: [INFO] AuthMe 5.6.0 build n.CUSTOM successfully enabled!
[01-20 09:19:32]: [FINE] Gimerbone logged in 192.168.100.84
[01-20 09:32:30]: [INFO] AuthMe 5.6.0-bCUSTOM disabled!
[01-20 10:03:21]: [INFO] Connection arguments loaded, Hikari ConnectionPool ready!
[01-20 10:03:22]: [INFO] MySQL setup finished
[01-20 10:03:22]: [INFO] No supported permissions system found! Permissions are disabled!
[01-20 10:03:22]: [INFO] AuthMe 5.6.0 build n.CUSTOM successfully enabled!
[01-20 10:03:43]: [FINE] The user Gimerbone has 3 accounts:
[01-20 10:03:43]: [FINE] §7§aGimerbone§7, eirangerz, yaminousagi.
[01-20 10:03:43]: [FINE] Gimerbone logged in 192.168.100.105
[01-20 10:14:02]: [INFO] AuthMe 5.6.0-bCUSTOM disabled!
[01-20 10:14:18]: [INFO] Connection arguments loaded, Hikari ConnectionPool ready!
[01-20 10:14:19]: [INFO] MySQL setup finished
[01-20 10:14:19]: [INFO] No supported permissions system found! Permissions are disabled!
[01-20 10:14:19]: [INFO] AuthMe 5.6.0 build n.CUSTOM successfully enabled!
[01-20 10:15:26]: [FINE] The user Gimerbone has 3 accounts:
[01-20 10:15:26]: [FINE] §7§aGimerbone§7, eirangerz, yaminousagi.
[01-20 10:15:26]: [FINE] Gimerbone logged in 192.168.100.105
[01-20 10:58:41]: [FINE] The user Gimerbone has 3 accounts:
[01-20 10:58:41]: [FINE] §7§aGimerbone§7, eirangerz, yaminousagi.
[01-20 10:58:41]: [FINE] Gimerbone logged in 192.168.100.105
[01-20 11:04:08]: [FINE] The user Gimerbone has 3 accounts:
[01-20 11:04:08]: [FINE] §7§aGimerbone§7, eirangerz, yaminousagi.
[01-20 11:04:08]: [FINE] Gimerbone logged in 192.168.100.105
[01-20 11:05:53]: [FINE] The user Gimerbone has 3 accounts:
[01-20 11:05:53]: [FINE] §7§aGimerbone§7, eirangerz, yaminousagi.
[01-20 11:05:53]: [FINE] Gimerbone logged in 192.168.100.105
[01-20 11:07:27]: [FINE] The user Gimerbone has 3 accounts:
[01-20 11:07:27]: [FINE] §7§aGimerbone§7, eirangerz, yaminousagi.
[01-20 11:07:27]: [FINE] Gimerbone logged in 192.168.100.105
[01-20 11:09:03]: [FINE] The user Gimerbone has 3 accounts:
[01-20 11:09:03]: [FINE] §7§aGimerbone§7, eirangerz, yaminousagi.
[01-20 11:09:03]: [FINE] Gimerbone logged in 192.168.100.105
[01-20 11:09:33]: [FINE] The user Gimerbone has 3 accounts:
[01-20 11:09:33]: [FINE] §7§aGimerbone§7, eirangerz, yaminousagi.
[01-20 11:09:33]: [FINE] Gimerbone logged in 192.168.100.105
[01-20 11:22:23]: [FINE] The user Gimerbone has 3 accounts:
[01-20 11:22:23]: [FINE] §7§aGimerbone§7, eirangerz, yaminousagi.
[01-20 11:22:23]: [FINE] Gimerbone logged in 192.168.100.105
[01-20 12:14:39]: [FINE] The user Gimerbone has 3 accounts:
[01-20 12:14:39]: [FINE] §7§aGimerbone§7, eirangerz, yaminousagi.
[01-20 12:14:39]: [FINE] Gimerbone logged in 192.168.100.105
[01-20 12:58:27]: [FINE] The user EiRanGerZ has 3 accounts:
[01-20 12:58:27]: [FINE] §7gimerbone, §aEiRanGerZ§7, yaminousagi.
[01-20 12:58:27]: [FINE] EiRanGerZ logged in 192.168.100.105
[01-20 13:01:57]: [FINE] The user Gimerbone has 3 accounts:
[01-20 13:01:57]: [FINE] §7§aGimerbone§7, eirangerz, yaminousagi.
[01-20 13:01:57]: [FINE] Gimerbone logged in 192.168.100.105
[01-20 13:03:16]: [FINE] The user Gimerbone has 3 accounts:
[01-20 13:03:16]: [FINE] §7§aGimerbone§7, eirangerz, yaminousagi.
[01-20 13:03:16]: [FINE] Gimerbone logged in 192.168.100.105
[01-20 13:05:15]: [FINE] The user Gimerbone has 3 accounts:
[01-20 13:05:15]: [FINE] §7§aGimerbone§7, eirangerz, yaminousagi.
[01-20 13:05:15]: [FINE] Gimerbone logged in 192.168.100.105
[01-20 13:19:48]: [FINE] The user EiRanGerZ has 3 accounts:
[01-20 13:19:48]: [FINE] §7gimerbone, §aEiRanGerZ§7, yaminousagi.
[01-20 13:19:48]: [FINE] EiRanGerZ logged in 192.168.100.105
[01-20 13:31:17]: [FINE] The user Gimerbone has 3 accounts:
[01-20 13:31:17]: [FINE] §7§aGimerbone§7, eirangerz, yaminousagi.
[01-20 13:31:17]: [FINE] Gimerbone logged in 192.168.100.105
[01-20 13:36:11]: [FINE] The user Gimerbone has 3 accounts:
[01-20 13:36:11]: [FINE] §7§aGimerbone§7, eirangerz, yaminousagi.
[01-20 13:36:11]: [FINE] Gimerbone logged in 192.168.100.105
[01-20 13:38:00]: [FINE] The user Gimerbone has 3 accounts:
[01-20 13:38:00]: [FINE] §7§aGimerbone§7, eirangerz, yaminousagi.
[01-20 13:38:00]: [FINE] Gimerbone logged in 192.168.100.105
[01-20 13:39:12]: [FINE] The user Gimerbone has 3 accounts:
[01-20 13:39:12]: [FINE] §7§aGimerbone§7, eirangerz, yaminousagi.
[01-20 13:39:12]: [FINE] Gimerbone logged in 192.168.100.105
[01-20 13:58:46]: [FINE] The user Gimerbone has 3 accounts:
[01-20 13:58:46]: [FINE] §7§aGimerbone§7, eirangerz, yaminousagi.
[01-20 13:58:46]: [FINE] Gimerbone logged in 192.168.100.105
[01-20 14:19:32]: [FINE] The user yaminousagi has 3 accounts:
[01-20 14:19:32]: [FINE] §7gimerbone, eirangerz, §ayaminousagi§7.
[01-20 14:19:32]: [FINE] yaminousagi logged in 192.168.100.105
[01-20 14:22:57]: [FINE] The user Gimerbone has 3 accounts:
[01-20 14:22:57]: [FINE] §7§aGimerbone§7, eirangerz, yaminousagi.
[01-20 14:22:57]: [FINE] Gimerbone logged in 192.168.100.105
[01-20 14:39:15]: [FINE] The user yaminousagi has 3 accounts:
[01-20 14:39:15]: [FINE] §7gimerbone, eirangerz, §ayaminousagi§7.
[01-20 14:39:15]: [FINE] yaminousagi logged in 192.168.100.105
[01-20 14:39:28]: [FINE] The user Gimerbone has 3 accounts:
[01-20 14:39:28]: [FINE] §7§aGimerbone§7, eirangerz, §ayaminousagi§7.
[01-20 14:39:28]: [FINE] Gimerbone logged in 192.168.100.105
[01-20 16:08:27]: [INFO] AuthMe 5.6.0-bCUSTOM disabled!
[01-21 02:35:47]: [INFO] Connection arguments loaded, Hikari ConnectionPool ready!
[01-21 02:35:47]: [INFO] MySQL setup finished
[01-21 02:35:47]: [INFO] No supported permissions system found! Permissions are disabled!
[01-21 02:35:48]: [INFO] AuthMe 5.6.0 build n.CUSTOM successfully enabled!
[01-21 02:44:37]: [FINE] The user Gimerbone has 3 accounts:
[01-21 02:44:37]: [FINE] §7§aGimerbone§7, eirangerz, yaminousagi.
[01-21 02:44:37]: [FINE] Gimerbone logged in 192.168.100.105
[01-21 02:53:59]: [FINE] The user Gimerbone has 3 accounts:
[01-21 02:53:59]: [FINE] §7§aGimerbone§7, eirangerz, yaminousagi.
[01-21 02:53:59]: [FINE] Gimerbone logged in 192.168.100.105
[01-21 03:09:43]: [FINE] The user Gimerbone has 3 accounts:
[01-21 03:09:43]: [FINE] §7§aGimerbone§7, eirangerz, yaminousagi.
[01-21 03:09:43]: [FINE] Gimerbone logged in 192.168.100.105
[01-21 03:15:13]: [FINE] The user Gimerbone has 3 accounts:
[01-21 03:15:13]: [FINE] §7§aGimerbone§7, eirangerz, yaminousagi.
[01-21 03:15:13]: [FINE] Gimerbone logged in 192.168.100.105
[01-21 03:16:01]: [FINE] The user Gimerbone has 3 accounts:
[01-21 03:16:01]: [FINE] §7§aGimerbone§7, eirangerz, yaminousagi.
[01-21 03:16:01]: [FINE] Gimerbone logged in 192.168.100.105
[01-21 03:29:00]: [FINE] The user Gimerbone has 3 accounts:
[01-21 03:29:00]: [FINE] §7§aGimerbone§7, eirangerz, yaminousagi.
[01-21 03:29:00]: [FINE] Gimerbone logged in 192.168.100.105
[01-21 03:37:31]: [FINE] The user Gimerbone has 3 accounts:
[01-21 03:37:31]: [FINE] §7§aGimerbone§7, eirangerz, yaminousagi.
[01-21 03:37:31]: [FINE] Gimerbone logged in 192.168.100.105
[01-21 03:45:18]: [FINE] The user Gimerbone has 3 accounts:
[01-21 03:45:18]: [FINE] §7§aGimerbone§7, eirangerz, yaminousagi.
[01-21 03:45:18]: [FINE] Gimerbone logged in 192.168.100.105
[01-21 03:45:48]: [FINE] The user Gimerbone has 3 accounts:
[01-21 03:45:48]: [FINE] §7§aGimerbone§7, eirangerz, yaminousagi.
[01-21 03:45:48]: [FINE] Gimerbone logged in 192.168.100.105
[01-21 03:47:59]: [INFO] AuthMe 5.6.0-bCUSTOM disabled!
[01-21 03:48:15]: [INFO] Connection arguments loaded, Hikari ConnectionPool ready!
[01-21 03:48:16]: [INFO] MySQL setup finished
[01-21 03:48:16]: [INFO] No supported permissions system found! Permissions are disabled!
[01-21 03:48:16]: [INFO] AuthMe 5.6.0 build n.CUSTOM successfully enabled!
[01-21 03:49:18]: [FINE] The user Gimerbone has 3 accounts:
[01-21 03:49:18]: [FINE] §7§aGimerbone§7, eirangerz, yaminousagi.
[01-21 03:49:18]: [FINE] Gimerbone logged in 192.168.100.105
[01-21 04:00:24]: [FINE] The user Gimerbone has 3 accounts:
[01-21 04:00:24]: [FINE] §7§aGimerbone§7, eirangerz, yaminousagi.
[01-21 04:00:24]: [FINE] Gimerbone logged in 192.168.100.105
[01-21 04:09:19]: [FINE] The user Gimerbone has 3 accounts:
[01-21 04:09:19]: [FINE] §7§aGimerbone§7, eirangerz, yaminousagi.
[01-21 04:09:19]: [FINE] Gimerbone logged in 192.168.100.105
[01-21 04:15:20]: [FINE] The user Gimerbone has 3 accounts:
[01-21 04:15:20]: [FINE] §7§aGimerbone§7, eirangerz, yaminousagi.
[01-21 04:15:20]: [FINE] Gimerbone logged in 192.168.100.105
[01-21 04:47:51]: [FINE] The user Gimerbone has 3 accounts:
[01-21 04:47:51]: [FINE] §7§aGimerbone§7, eirangerz, yaminousagi.
[01-21 04:47:51]: [FINE] Gimerbone logged in 192.168.100.105
[01-21 04:56:19]: [FINE] The user Gimerbone has 3 accounts:
[01-21 04:56:19]: [FINE] §7§aGimerbone§7, eirangerz, yaminousagi.
[01-21 04:56:19]: [FINE] Gimerbone logged in 192.168.100.105
[01-21 06:17:39]: [FINE] The user Gimerbone has 3 accounts:
[01-21 06:17:39]: [FINE] §7§aGimerbone§7, eirangerz, yaminousagi.
[01-21 06:17:39]: [FINE] Gimerbone logged in 192.168.100.105
[01-21 06:25:23]: [FINE] The user Gimerbone has 3 accounts:
[01-21 06:25:23]: [FINE] §7§aGimerbone§7, eirangerz, yaminousagi.
[01-21 06:25:23]: [FINE] Gimerbone logged in 192.168.100.105
[01-21 06:49:14]: [FINE] BaleeXjaa registered 192.168.100.105
[01-21 06:49:14]: [FINE] The user BaleeXjaa has 4 accounts:
[01-21 06:49:14]: [FINE] §7gimerbone, eirangerz, yaminousagi, §aBaleeXjaa§7.
[01-21 06:49:14]: [FINE] BaleeXjaa logged in 192.168.100.105
[01-21 06:49:31]: [FINE] The user EiRanGerZ has 4 accounts:
[01-21 06:49:31]: [FINE] §7gimerbone, §aEiRanGerZ§7, yaminousagi, §aBaleeXjaa§7.
[01-21 06:49:31]: [FINE] EiRanGerZ logged in 192.168.100.105
[01-21 06:50:50]: [FINE] ElPunci registered 192.168.100.105
[01-21 06:50:50]: [FINE] The user ElPunci has 5 accounts:
[01-21 06:50:50]: [FINE] §7gimerbone, eirangerz, yaminousagi, §aBaleeXjaa§7, §aElPunci§7.
[01-21 06:50:50]: [FINE] ElPunci logged in 192.168.100.105
[01-21 06:51:27]: [FINE] The user EiRanGerZ has 5 accounts:
[01-21 06:51:27]: [FINE] §7gimerbone, §aEiRanGerZ§7, yaminousagi, §aBaleeXjaa§7, §aElPunci§7.
[01-21 06:51:27]: [FINE] EiRanGerZ logged in 192.168.100.105
[01-21 06:55:08]: [FINE] The user Gimerbone has 5 accounts:
[01-21 06:55:08]: [FINE] §7§aGimerbone§7, eirangerz, yaminousagi, baleexjaa, elpunci.
[01-21 06:55:08]: [FINE] Gimerbone logged in 192.168.100.105
[01-21 08:32:34]: [FINE] The user ElPunci has 5 accounts:
[01-21 08:32:34]: [FINE] §7gimerbone, eirangerz, yaminousagi, baleexjaa, §aElPunci§7.
[01-21 08:32:34]: [FINE] ElPunci logged in 192.168.100.105
[01-21 10:21:40]: [FINE] The user ElPunci has 5 accounts:
[01-21 10:21:40]: [FINE] §7gimerbone, eirangerz, yaminousagi, baleexjaa, §aElPunci§7.
[01-21 10:21:40]: [FINE] ElPunci logged in 192.168.100.105
[01-21 10:26:36]: [FINE] The user Gimerbone has 5 accounts:
[01-21 10:26:36]: [FINE] §7§aGimerbone§7, eirangerz, yaminousagi, baleexjaa, elpunci.
[01-21 10:26:36]: [FINE] Gimerbone logged in 192.168.100.105
[01-21 13:24:08]: [FINE] The user Gimerbone has 5 accounts:
[01-21 13:24:08]: [FINE] §7§aGimerbone§7, eirangerz, yaminousagi, baleexjaa, elpunci.
[01-21 13:24:08]: [FINE] Gimerbone logged in 192.168.100.105
[01-21 14:25:37]: [FINE] The user Gimerbone has 5 accounts:
[01-21 14:25:37]: [FINE] §7§aGimerbone§7, eirangerz, yaminousagi, baleexjaa, elpunci.
[01-21 14:25:37]: [FINE] Gimerbone logged in 192.168.100.105
[01-21 15:26:16]: [INFO] AuthMe 5.6.0-bCUSTOM disabled!
[01-22 01:15:36]: [INFO] Connection arguments loaded, Hikari ConnectionPool ready!
[01-22 01:15:37]: [INFO] MySQL setup finished
[01-22 01:15:37]: [INFO] No supported permissions system found! Permissions are disabled!
[01-22 01:15:37]: [INFO] AuthMe 5.6.0 build n.CUSTOM successfully enabled!
[01-22 01:36:37]: [FINE] The user Gimerbone has 5 accounts:
[01-22 01:36:37]: [FINE] §7§aGimerbone§7, eirangerz, yaminousagi, baleexjaa, elpunci.
[01-22 01:36:37]: [FINE] Gimerbone logged in 192.168.100.105
[01-22 02:55:12]: [FINE] The user Gimerbone has 5 accounts:
[01-22 02:55:12]: [FINE] §7§aGimerbone§7, eirangerz, yaminousagi, baleexjaa, elpunci.
[01-22 02:55:12]: [FINE] Gimerbone logged in 192.168.100.105
[01-22 08:59:16]: [FINE] The user BaleeXjaa has 5 accounts:
[01-22 08:59:16]: [FINE] §7gimerbone, eirangerz, yaminousagi, §aBaleeXjaa§7, elpunci.
[01-22 08:59:16]: [FINE] BaleeXjaa logged in 192.168.100.105
[01-22 08:59:28]: [FINE] The user EiRanGerZ has 5 accounts:
[01-22 08:59:28]: [FINE] §7gimerbone, §aEiRanGerZ§7, yaminousagi, baleexjaa, elpunci.
[01-22 08:59:28]: [FINE] EiRanGerZ logged in 192.168.100.105
[01-22 09:00:52]: [FINE] BaleeXjaa used the wrong password
[01-22 09:01:02]: [FINE] The user BaleeXjaa has 5 accounts:
[01-22 09:01:02]: [FINE] §7gimerbone, eirangerz, yaminousagi, §aBaleeXjaa§7, elpunci.
[01-22 09:01:02]: [FINE] BaleeXjaa logged in 192.168.100.105
[01-22 12:30:54]: [FINE] DrtCoffee registered 192.168.100.105
[01-22 12:30:54]: [FINE] The user DrtCoffee has 6 accounts:
[01-22 12:30:54]: [FINE] §7gimerbone, eirangerz, yaminousagi, baleexjaa, elpunci, §aDrtCoffee§7.
[01-22 12:30:54]: [FINE] DrtCoffee logged in 192.168.100.105
[01-22 12:32:12]: [FINE] The user DrtCoffee has 6 accounts:
[01-22 12:32:12]: [FINE] §7gimerbone, eirangerz, yaminousagi, baleexjaa, elpunci, §aDrtCoffee§7.
[01-22 12:32:12]: [FINE] DrtCoffee logged in 192.168.100.105
[01-22 12:33:56]: [FINE] The user DrtCoffee has 6 accounts:
[01-22 12:33:56]: [FINE] §7gimerbone, eirangerz, yaminousagi, baleexjaa, elpunci, §aDrtCoffee§7.
[01-22 12:33:56]: [FINE] DrtCoffee logged in 192.168.100.105
[01-22 12:35:06]: [FINE] The user DrtCoffee has 6 accounts:
[01-22 12:35:06]: [FINE] §7gimerbone, eirangerz, yaminousagi, baleexjaa, elpunci, §aDrtCoffee§7.
[01-22 12:35:06]: [FINE] DrtCoffee logged in 192.168.100.105
[01-22 12:35:34]: [FINE] The user DrtCoffee has 6 accounts:
[01-22 12:35:34]: [FINE] §7gimerbone, eirangerz, yaminousagi, baleexjaa, elpunci, §aDrtCoffee§7.
[01-22 12:35:34]: [FINE] DrtCoffee logged in 192.168.100.105
[01-22 12:55:20]: [FINE] The user DrtCoffee has 6 accounts:
[01-22 12:55:20]: [FINE] §7gimerbone, eirangerz, yaminousagi, baleexjaa, elpunci, §aDrtCoffee§7.
[01-22 12:55:20]: [FINE] DrtCoffee logged in 192.168.100.105
[01-22 13:12:21]: [FINE] The user Gimerbone has 6 accounts:
[01-22 13:12:21]: [FINE] §7§aGimerbone§7, eirangerz, yaminousagi, baleexjaa, elpunci, drtcoffee.
[01-22 13:12:21]: [FINE] Gimerbone logged in 192.168.100.105
[01-22 15:25:37]: [INFO] AuthMe 5.6.0-bCUSTOM disabled!
[01-22 23:42:48]: [INFO] Connection arguments loaded, Hikari ConnectionPool ready!
[01-22 23:42:49]: [INFO] MySQL setup finished
[01-22 23:42:49]: [INFO] No supported permissions system found! Permissions are disabled!
[01-22 23:42:49]: [INFO] AuthMe 5.6.0 build n.CUSTOM successfully enabled!
[01-22 23:49:47]: [FINE] The user Gimerbone has 6 accounts:
[01-22 23:49:47]: [FINE] §7§aGimerbone§7, eirangerz, yaminousagi, baleexjaa, elpunci, drtcoffee.
[01-22 23:49:47]: [FINE] Gimerbone logged in 192.168.100.105
[01-23 00:50:36]: [FINE] The user Gimerbone has 6 accounts:
[01-23 00:50:36]: [FINE] §7§aGimerbone§7, eirangerz, yaminousagi, baleexjaa, elpunci, drtcoffee.
[01-23 00:50:36]: [FINE] Gimerbone logged in 192.168.100.105
[01-23 01:06:19]: [FINE] The user Gimerbone has 6 accounts:
[01-23 01:06:19]: [FINE] §7§aGimerbone§7, eirangerz, yaminousagi, baleexjaa, elpunci, drtcoffee.
[01-23 01:06:19]: [FINE] Gimerbone logged in 192.168.100.105
[01-23 04:19:01]: [FINE] The user yaminousagi has 6 accounts:
[01-23 04:19:01]: [FINE] §7gimerbone, eirangerz, §ayaminousagi§7, baleexjaa, elpunci, drtcoffee.
[01-23 04:19:01]: [FINE] yaminousagi logged in 192.168.100.105
[01-23 04:44:46]: [FINE] The user BaleeXjaa has 6 accounts:
[01-23 04:44:46]: [FINE] §7gimerbone, eirangerz, yaminousagi, §aBaleeXjaa§7, elpunci, drtcoffee.
[01-23 04:44:46]: [FINE] BaleeXjaa logged in 192.168.100.105
[01-23 04:46:05]: [FINE] The user yaminousagi has 6 accounts:
[01-23 04:46:05]: [FINE] §7gimerbone, eirangerz, §ayaminousagi§7, baleexjaa, elpunci, drtcoffee.
[01-23 04:46:05]: [FINE] yaminousagi logged in 192.168.100.105
[01-23 05:11:53]: [FINE] The user yaminousagi has 6 accounts:
[01-23 05:11:53]: [FINE] §7gimerbone, eirangerz, §ayaminousagi§7, baleexjaa, elpunci, drtcoffee.
[01-23 05:11:53]: [FINE] yaminousagi logged in 192.168.100.105
[01-23 07:42:27]: [FINE] The user yaminousagi has 6 accounts:
[01-23 07:42:27]: [FINE] §7gimerbone, eirangerz, §ayaminousagi§7, baleexjaa, elpunci, drtcoffee.
[01-23 07:42:27]: [FINE] yaminousagi logged in 192.168.100.105
[01-23 10:17:59]: [FINE] The user Gimerbone has 6 accounts:
[01-23 10:17:59]: [FINE] §7§aGimerbone§7, eirangerz, yaminousagi, baleexjaa, elpunci, drtcoffee.
[01-23 10:17:59]: [FINE] Gimerbone logged in 192.168.100.105
[01-23 10:28:27]: [FINE] The user Gimerbone has 6 accounts:
[01-23 10:28:27]: [FINE] §7§aGimerbone§7, eirangerz, yaminousagi, baleexjaa, elpunci, drtcoffee.
[01-23 10:28:27]: [FINE] Gimerbone logged in 192.168.100.105
[01-23 10:54:01]: [FINE] The user BaleeXjaa has 6 accounts:
[01-23 10:54:01]: [FINE] §7gimerbone, eirangerz, yaminousagi, §aBaleeXjaa§7, elpunci, drtcoffee.
[01-23 10:54:01]: [FINE] BaleeXjaa logged in 192.168.100.105
[01-23 12:03:48]: [FINE] The user Gimerbone has 6 accounts:
[01-23 12:03:48]: [FINE] §7§aGimerbone§7, eirangerz, yaminousagi, baleexjaa, elpunci, drtcoffee.
[01-23 12:03:48]: [FINE] Gimerbone logged in 192.168.100.105
[01-23 12:17:44]: [FINE] The user Gimerbone has 6 accounts:
[01-23 12:17:44]: [FINE] §7§aGimerbone§7, eirangerz, yaminousagi, baleexjaa, elpunci, drtcoffee.
[01-23 12:17:44]: [FINE] Gimerbone logged in 192.168.100.105
[01-23 12:20:30]: [FINE] The user EiRanGerZ has 6 accounts:
[01-23 12:20:30]: [FINE] §7gimerbone, §aEiRanGerZ§7, yaminousagi, baleexjaa, elpunci, drtcoffee.
[01-23 12:20:30]: [FINE] EiRanGerZ logged in 192.168.100.105
[01-23 12:21:10]: [FINE] The user EiRanGerZ has 6 accounts:
[01-23 12:21:10]: [FINE] §7gimerbone, §aEiRanGerZ§7, yaminousagi, baleexjaa, elpunci, drtcoffee.
[01-23 12:21:10]: [FINE] EiRanGerZ logged in 192.168.100.105
[01-23 12:22:36]: [FINE] The user EiRanGerZ has 6 accounts:
[01-23 12:22:36]: [FINE] §7gimerbone, §aEiRanGerZ§7, yaminousagi, baleexjaa, elpunci, drtcoffee.
[01-23 12:22:36]: [FINE] EiRanGerZ logged in 192.168.100.105
[01-23 12:39:18]: [FINE] The user yaminousagi has 6 accounts:
[01-23 12:39:18]: [FINE] §7gimerbone, eirangerz, §ayaminousagi§7, baleexjaa, elpunci, drtcoffee.
[01-23 12:39:18]: [FINE] yaminousagi logged in 192.168.100.105
[01-23 13:03:49]: [FINE] The user Gimerbone has 6 accounts:
[01-23 13:03:49]: [FINE] §7§aGimerbone§7, eirangerz, yaminousagi, baleexjaa, elpunci, drtcoffee.
[01-23 13:03:49]: [FINE] Gimerbone logged in 192.168.100.105
[01-23 13:06:55]: [FINE] The user yaminousagi has 6 accounts:
[01-23 13:06:55]: [FINE] §7gimerbone, eirangerz, §ayaminousagi§7, baleexjaa, elpunci, drtcoffee.
[01-23 13:06:55]: [FINE] yaminousagi logged in 192.168.100.105
[01-23 13:08:15]: [FINE] The user yaminousagi has 6 accounts:
[01-23 13:08:15]: [FINE] §7gimerbone, eirangerz, §ayaminousagi§7, baleexjaa, elpunci, drtcoffee.
[01-23 13:08:15]: [FINE] yaminousagi logged in 192.168.100.105
[01-23 13:09:05]: [FINE] The user Gimerbone has 6 accounts:
[01-23 13:09:05]: [FINE] §7§aGimerbone§7, eirangerz, yaminousagi, baleexjaa, elpunci, drtcoffee.
[01-23 13:09:05]: [FINE] Gimerbone logged in 192.168.100.105
[01-23 13:11:51]: [FINE] The user yaminousagi has 6 accounts:
[01-23 13:11:51]: [FINE] §7§aGimerbone§7, eirangerz, §ayaminousagi§7, baleexjaa, elpunci, drtcoffee.
[01-23 13:11:51]: [FINE] yaminousagi logged in 192.168.100.105
[01-23 13:16:48]: [FINE] The user BaleeXjaa has 6 accounts:
[01-23 13:16:48]: [FINE] §7gimerbone, eirangerz, yaminousagi, §aBaleeXjaa§7, elpunci, drtcoffee.
[01-23 13:16:48]: [FINE] BaleeXjaa logged in 192.168.100.105
[01-23 14:09:02]: [FINE] The user yaminousagi has 6 accounts:
[01-23 14:09:02]: [FINE] §7gimerbone, eirangerz, §ayaminousagi§7, baleexjaa, elpunci, drtcoffee.
[01-23 14:09:02]: [FINE] yaminousagi logged in 192.168.100.105
[01-23 14:10:47]: [FINE] The user yaminousagi has 6 accounts:
[01-23 14:10:47]: [FINE] §7gimerbone, eirangerz, §ayaminousagi§7, baleexjaa, elpunci, drtcoffee.
[01-23 14:10:47]: [FINE] yaminousagi logged in 192.168.100.105
[01-23 15:31:08]: [INFO] AuthMe 5.6.0-bCUSTOM disabled!
[01-24 02:41:23]: [INFO] Connection arguments loaded, Hikari ConnectionPool ready!
[01-24 02:41:24]: [INFO] MySQL setup finished
[01-24 02:41:24]: [INFO] No supported permissions system found! Permissions are disabled!
[01-24 02:41:24]: [INFO] AuthMe 5.6.0 build n.CUSTOM successfully enabled!
[01-24 06:27:13]: [FINE] The user Gimerbone has 6 accounts:
[01-24 06:27:13]: [FINE] §7§aGimerbone§7, eirangerz, yaminousagi, baleexjaa, elpunci, drtcoffee.
[01-24 06:27:13]: [FINE] Gimerbone logged in 192.168.100.105
[01-24 07:11:39]: [FINE] The user BaleeXjaa has 6 accounts:
[01-24 07:11:39]: [FINE] §7gimerbone, eirangerz, yaminousagi, §aBaleeXjaa§7, elpunci, drtcoffee.
[01-24 07:11:39]: [FINE] BaleeXjaa logged in 192.168.100.105
[01-24 10:49:11]: [FINE] The user Gimerbone has 6 accounts:
[01-24 10:49:11]: [FINE] §7§aGimerbone§7, eirangerz, yaminousagi, baleexjaa, elpunci, drtcoffee.
[01-24 10:49:11]: [FINE] Gimerbone logged in 192.168.100.105
[01-24 10:50:36]: [INFO] AuthMe 5.6.0-bCUSTOM disabled!
[01-24 10:50:53]: [INFO] Connection arguments loaded, Hikari ConnectionPool ready!
[01-24 10:50:53]: [INFO] MySQL setup finished
[01-24 10:50:54]: [INFO] No supported permissions system found! Permissions are disabled!
[01-24 10:50:54]: [INFO] AuthMe 5.6.0 build n.CUSTOM successfully enabled!
[01-24 10:55:37]: [INFO] AuthMe 5.6.0-bCUSTOM disabled!
[01-24 10:56:19]: [INFO] Connection arguments loaded, Hikari ConnectionPool ready!
[01-24 10:56:20]: [INFO] MySQL setup finished
[01-24 10:56:20]: [INFO] No supported permissions system found! Permissions are disabled!
[01-24 10:56:20]: [INFO] AuthMe 5.6.0 build n.CUSTOM successfully enabled!
[01-24 10:58:24]: [FINE] The user Gimerbone has 6 accounts:
[01-24 10:58:24]: [FINE] §7§aGimerbone§7, eirangerz, yaminousagi, baleexjaa, elpunci, drtcoffee.
[01-24 10:58:24]: [FINE] Gimerbone logged in 192.168.100.105
[01-24 12:19:52]: [FINE] The user Gimerbone has 6 accounts:
[01-24 12:19:52]: [FINE] §7§aGimerbone§7, eirangerz, yaminousagi, baleexjaa, elpunci, drtcoffee.
[01-24 12:19:52]: [FINE] Gimerbone logged in 192.168.100.105
[01-24 12:21:16]: [FINE] The user Gimerbone has 6 accounts:
[01-24 12:21:16]: [FINE] §7§aGimerbone§7, eirangerz, yaminousagi, baleexjaa, elpunci, drtcoffee.
[01-24 12:21:16]: [FINE] Gimerbone logged in 192.168.100.105
[01-24 12:21:55]: [FINE] The user Gimerbone has 6 accounts:
[01-24 12:21:55]: [FINE] §7§aGimerbone§7, eirangerz, yaminousagi, baleexjaa, elpunci, drtcoffee.
[01-24 12:21:55]: [FINE] Gimerbone logged in 192.168.100.105
[01-24 13:45:35]: [FINE] The user Gimerbone has 6 accounts:
[01-24 13:45:35]: [FINE] §7§aGimerbone§7, eirangerz, yaminousagi, baleexjaa, elpunci, drtcoffee.
[01-24 13:45:35]: [FINE] Gimerbone logged in 192.168.100.105
[01-24 13:54:04]: [FINE] The user Gimerbone has 6 accounts:
[01-24 13:54:04]: [FINE] §7§aGimerbone§7, eirangerz, yaminousagi, baleexjaa, elpunci, drtcoffee.
[01-24 13:54:04]: [FINE] Gimerbone logged in 192.168.100.105
[01-24 16:04:27]: [INFO] AuthMe 5.6.0-bCUSTOM disabled!
[01-24 23:27:12]: [INFO] Connection arguments loaded, Hikari ConnectionPool ready!
[01-24 23:27:12]: [INFO] MySQL setup finished
[01-24 23:27:12]: [INFO] No supported permissions system found! Permissions are disabled!
[01-24 23:27:13]: [INFO] AuthMe 5.6.0 build n.CUSTOM successfully enabled!
[01-25 07:09:24]: [FINE] The user Gimerbone has 6 accounts:
[01-25 07:09:24]: [FINE] §7§aGimerbone§7, eirangerz, yaminousagi, baleexjaa, elpunci, drtcoffee.
[01-25 07:09:24]: [FINE] Gimerbone logged in 192.168.100.105
[01-25 07:27:31]: [FINE] The user BaleeXjaa has 6 accounts:
[01-25 07:27:31]: [FINE] §7gimerbone, eirangerz, yaminousagi, §aBaleeXjaa§7, elpunci, drtcoffee.
[01-25 07:27:31]: [FINE] BaleeXjaa logged in 192.168.100.105
[01-25 16:15:51]: [INFO] AuthMe 5.6.0-bCUSTOM disabled!
[01-26 03:32:08]: [INFO] Connection arguments loaded, Hikari ConnectionPool ready!
[01-26 03:32:09]: [INFO] MySQL setup finished
[01-26 03:32:09]: [INFO] No supported permissions system found! Permissions are disabled!
[01-26 03:32:10]: [INFO] AuthMe 5.6.0 build n.CUSTOM successfully enabled!
[01-26 06:17:39]: [FINE] The user Gimerbone has 6 accounts:
[01-26 06:17:39]: [FINE] §7§aGimerbone§7, eirangerz, yaminousagi, baleexjaa, elpunci, drtcoffee.
[01-26 06:17:39]: [FINE] Gimerbone logged in 192.168.100.105
[01-26 16:08:29]: [INFO] AuthMe 5.6.0-bCUSTOM disabled!
[01-28 12:22:33]: [INFO] Connection arguments loaded, Hikari ConnectionPool ready!
[01-28 12:22:33]: [INFO] MySQL setup finished
[01-28 12:22:34]: [INFO] No supported permissions system found! Permissions are disabled!
[01-28 12:22:34]: [INFO] AuthMe 5.6.0 build n.CUSTOM successfully enabled!
[01-28 15:00:18]: [INFO] AuthMe 5.6.0-bCUSTOM disabled!
[01-29 04:51:20]: [INFO] Connection arguments loaded, Hikari ConnectionPool ready!
[01-29 04:51:21]: [INFO] MySQL setup finished
[01-29 04:51:21]: [INFO] No supported permissions system found! Permissions are disabled!
[01-29 04:51:21]: [INFO] AuthMe 5.6.0 build n.CUSTOM successfully enabled!
[01-29 10:55:05]: [FINE] Gimerbone logged in 192.168.100.84
[01-29 16:13:08]: [INFO] AuthMe 5.6.0-bCUSTOM disabled!
[01-30 09:17:06]: [INFO] Connection arguments loaded, Hikari ConnectionPool ready!
[01-30 09:17:07]: [INFO] MySQL setup finished
[01-30 09:17:07]: [INFO] No supported permissions system found! Permissions are disabled!
[01-30 09:17:08]: [INFO] AuthMe 5.6.0 build n.CUSTOM successfully enabled!
[01-30 16:04:10]: [INFO] AuthMe 5.6.0-bCUSTOM disabled!
